package.json

```diff
@@ -18,9 +18,14 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme": "^3.8.0",
+    "enzyme-adapter-react-16": "^1.7.1",
+    "enzyme-to-json": "^3.3.5",
     "gh-pages": "^1.2.0"
   },
   "homepage": "http://adagold.github.io/inspiration-board"
 }
+
+"jest": {
+  "snapshotSerializers": ["enzyme-to-json/serializer"]
+}
```

Comment:
You made the `jest` section outside of the big object here, when really it needs to be a sibling to `homepage`